pipeline/sj_luigi_luigi.py

```diff
@@ -110,8 +110,9 @@ def run(self):
         # creates EmailMessage object
         msg = EmailMessage()
         msg["Subject"] = self.subject
+        msg["Date"] = datetime.now() -+ timedelta(hours=8)
         msg["From"] = email().email_from
-        msg["To"] = ", ".join(self.receipients)
+        msg["To"] = email().email_to.split(", ")
```

Comment:
how you put your email_to?
email_to="email1@example.com, email2@example.com"
or
email_to=email1@example.com, email2@example.com

Reply:
the second one, email_to=[email1@example.com](mailto:email1@example.com), [email2@example.com](mailto:email2@example.com)

Reply:
ok. actually you dont need to use split, what if someone else write it without space after 'comma', email_to=email1@example.com,email2@example.com

do you think your split still work?

Reply:
if you used luigi.Parameter(), the luigi will treat it as str type, so you just need to msg["To"]=email().email_to will do.

Reply:
no need to change anything

Reply:
Ok I will have a try, thanks

Reply:
Now I got it! Tysm